.babelrc

```
@@ -0,0 +1,17 @@
+{
+	"plugins": [
+		"@babel/plugin-transform-runtime",
+		[
+			"transform-react-remove-prop-types",
+			{
+				mode: "remove",
+				removeImport: true
+			}
+		]
+	],
+	"presets": [
+		"@babel/preset-env",
```

```suggestion
		"@babel/preset-env",
		{
			"targets": {
				"esmodules": true
			}
		},
```

I would suggest outputting ESModules instead of CommonJS. I think this is right? And then add a `"module": ...` to package.json alongside `"main": ...`

The [Webpack tree shaking guide](https://webpack.js.org/guides/tree-shaking/#conclusion) seems to suggest that ESModule-style imports are required for any tree shaking to occur? But maybe I'm reading this wrong:

> Use ES2015 module syntax (i.e. import and export).
> Ensure no compilers transform your ES2015 module syntax into CommonJS modules (this is the default behavior of the popular Babel preset @babel/preset-env - see the documentation for more details).